### PR TITLE
Remove Typos

### PR DESCRIPTION
**Pull request by nirmalsinghania2008**

### CODE REVIEW

**Comment by nirmalsinghania2008** [comment]
I just removed the typos.why is this failing? 
can someone guide me?

File: sml/include/sml_log.h [hunk 1]
```diff
@@ -35,11 +35,11 @@ extern "C" {
...
-    SML_LOG_LEVEL_WARNING = 1 << 2, /**< Show warning messages. */
-    SML_LOG_LEVEL_ERROR = 1 << 3, /**< Show error messages. */
-    SML_LOG_LEVEL_CRITICAL = 1 << 4, /**< Show critical messages. */
-    SML_LOG_LEVEL_ALL = (SML_LOG_LEVEL_DEBUG | SML_LOG_LEVEL_INFO | SML_LOG_LEVEL_WARNING | SML_LOG_LEVEL_ERROR | SML_LOG_LEVEL_CRITICAL) /**< Show all messages. */
+        SML_LOG_LEVEL_INFO = 1 << 1,     /**< Show info messages. */
+        SML_LOG_LEVEL_WARNING = 1 << 2, /**< Show warning messages. */
+        SML_LOG_LEVEL_ERROR = 1 << 3, /**< Show error messages. */
+        SML_LOG_LEVEL_CRITICAL = 1 << 4, /**< Show critical messages. */
+        SML_LOG_LEVEL_ALL = (SML_LOG_LEVEL_DEBUG | SML_LOG_LEVEL_INFO | SML_LOG_LEVEL_WARNING | SML_LOG_LEVEL_ERROR | SML_LOG_LEVEL_CRITICAL) /**< Show all messages. */
 };     /**< The types of log levels */
```
Comment (otaviobp) [inline sml/include/sml_log.h:43]: This report is an issue with older versions of uncrustify. If you use uncrustify with version >= 0.62, this won't be reported. In Soletta we have some enums with the style you are proposing, but we preferring to use the style proposed by a newer uncrustify.

**Comment by otaviobp** [comment]
This bug was fixed in pull req #72 but was not merged yet.

**Comment by nirmalsinghania2008** [comment]
now,what i have to do to get the typos removed?

**Comment by otaviobp** [comment]
I just merged the fix. please rebase, fix the problem I commented, build and test it, then send a version 2. Take a look at https://github.com/solettaproject/soletta/wiki/Contributing to see how to send a v2.

**Pull request closed without merging**